apps/web/env.ts

```diff
@@ -12,6 +12,10 @@ export const env = createEnv({
     GOOGLE_CLIENT_SECRET: z.string().min(1),
     GOOGLE_ENCRYPT_SECRET: z.string(),
     GOOGLE_ENCRYPT_SALT: z.string(),
+    MICROSOFT_CLIENT_ID: z.string().min(1),
+    MICROSOFT_CLIENT_SECRET: z.string().min(1),
+    MICROSOFT_ENCRYPT_SECRET: z.string(),
+    MICROSOFT_ENCRYPT_SALT: z.string(),
```

Comment:
We can make these optional so that people cna run the project without setting Microsoft keys